x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/RequestIndexFilteringIT.java

```diff
@@ -87,6 +88,9 @@ protected String from(String... indexName) {
 
     @Override
     public Map<String, Object> runEsql(RestEsqlTestCase.RequestObjectBuilder requestObject) throws IOException {
+        if (requestObject.allowPartialResults() != null) {
+            assumeTrue("require allow_partial_results on local cluster", Clusters.localClusterVersion().equals(Version.V_8_19_0));
```

Comment:
Not sure I understand - why it's only 8.19? What about 9.1 and after that?

Reply:
Oops - We should use caps here . See https://github.com/elastic/elasticsearch/pull/125060/commits/cbb208ae63bc4e48f9247a139120b389170c4940. Thanks!